site/includes/footer.php

```diff
@@ -1,6 +1,34 @@
-        <footer class="bg-dark">
-            <p class="h2 mb-4">Open Game Data</p>
-            <img class="logo" src="assets/images/logos/FieldDay-logo.png" alt="Field Day">
+        <footer>
+            <div class="row">
+                <div id="footer-ogd" class="col mt-5 mb-5">
+                    <img class="ogd-logo" src="assets/images/logos/OpenGameData-logo-black.png" alt="Field Day">
+                    <div class="fd-logo-wrapper mt-2">
+                       <a href="https://fielddaylab.wisc.edu/" target="_blank" rel="noopener"><em class="me-2">A Project by:</em> <img src="assets/images/logos/FieldDay-logo-black.png" alt="Field Day"></a>
+                    </div>
+                    <p class="mt-5">
+                        We have built an open-source and community-maintained data storage and processing pipeline for educational game data.
+                        From the logging libraries that studios can integrate into their games, all the way to the final visualizations that allow
+                        design researchers to build new theory, we are thinking about modularity, scalability and performance.
+                    </p>
+                    <a class="btn btn-signup mt-4" href="https://forms.gle/z9HGvu5Kmsuf6xDD8" target="_blank" rel="nofollow noopener">Sign up to receive communications<i class="bi bi-chevron-right ms-5"></i></a>
+                </div>
+                <div class="col-1 vr-wrapper">
+                    <div class="vr"></div>
+                </div>
+                <div id="footer-uw-wcer" class="col">
+                    <div class="uw-logo-wrapper text-center">
+                        <a href="https://wcer.wisc.edu/" target="_blank" rel="noopener"><img src="assets/images/logos/uw-madison-logo.png" alt="University of Wisconsin Madison"></a>
+                    </div>
+                    <p class="mt-5">
+                        <strong>Field Day</strong> is based at the Wisconsin Center for Educational Research at the University of Wisconsin - Madison. We make learning
+                        games that bring contemporary research to the public, and we use the game data to understand how people learn.
+                    </p>
+                    <p class="mt-5"><strong>Funding Provided By:</strong></p>
+                    <img src="assets/images/logos/nsf-logo.png" alt="National Science Foundation">
+                    <img class="ms-3" src="assets/images/logos/the-learning-agency-logo.png" alt="The Learning Agency">
```

Comment:
I think it is best to try to keep these three logos on the same line as much as possible as the screen narrows. I suggest going with `ms-1 ms-lg-3` if you agree. They are closer together than in the mockup at the md width, but I think that is preferable to wrapping. This would apply to the Tools Competition logo as well.